utils/exec.go

```diff
@@ -4,20 +4,32 @@ import (
 	"bufio"
 	"bytes"
 	"fmt"
-	log "github.com/Sirupsen/logrus"
 	"io"
+	"io/ioutil"
 	"os"
 	"os/exec"
 	"runtime"
 	"strings"
 	"syscall"
 	"time"
+
+	log "github.com/Sirupsen/logrus"
 )
 
 const (
-	TimeStampLayout = "2006-01-02T15:04:05.000000-07:00"
+	TimeStampLayout          = "2006-01-02T15:04:05.000000-07:00"
+	TimeLayoutYYYYMMDDHHMMSS = "20060102150405"
+	DefaultThresholdLines    = 10
+	DefaultThresholdTime     = 10
+	DefaultThresholdBytes    = 500
```

Comment:
Not sure this is the best place for these constants, which are very specific to the continuous report run command

Reply:
yes, it previously was out of this location. But the idea for this is about the possibility of future uses; i mean, a continuous command but with different management for chunked responses.